src/payment/data/service.js

```diff
@@ -56,6 +56,13 @@ export async function getCaptureKey() {
   return data;
 }
 
+export async function getClientSecret() {
```

Comment:
Is there any difference between `getClientSecret()` and `getCaptureKey()`? If no, should we have 2?

Reply:
As of now no since the backend can only return either the capture key or the client secret.  created client secret separately so there was a clear distinction between stripe a cybersource since the capture key is used for things like microform status and the timeout warnings on the form.  I think its useful to have both for when we can have cybersource and stripe active at the same time but I can remove for now until necessary.

Reply:
Sounds good to keep it